server/.env

```diff
@@ -1 +1,3 @@
 TEAM_MEMBERS="Aecio, Bonnie, Dhruv, Hakeem, Thiago"
+AWS_ACCESS_KEY_ID = "AKIAY24KUFXNQW3YNBNI"
+AWS_SECRET_ACCESS_KEY = "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"
```

Comment:
The .env file should be ignored by git now. These entries can't be in the remote repository for security reasons.

Reply:
I know, that is why I said in the notes, that it is temporary. I intent to discuss the solution for it in today's standup.